pkg/lib/ratelimit/ratelimits.go

```diff
@@ -183,6 +203,26 @@ func (n RateLimit) resolvePerUser(cfg *config.AppConfig) *config.RateLimitConfig
 	return nil
 }
 
+func (n RateLimit) resolvePerTarget(cfg *config.AppConfig, featureCfg *config.FeatureConfig) *config.RateLimitConfig {
+	get := func(c *config.RateLimitConfig, featureConfig *config.RateLimitConfig) *config.RateLimitConfig {
+		effectiveConfig := c
+		if featureConfig != nil && featureConfig.Rate() < c.Rate() {
```

Comment:
Should we check featureConfig is enabled instead of checking `!= nil`?

Reply:
Same as https://github.com/authgear/authgear-server/pull/5251#discussion_r2174200305